packages/odyssey-react-mui/src/Autocomplete.tsx

```diff
@@ -474,6 +474,7 @@ const Autocomplete = <
       onInputChange={onInputChange}
       onFocus={onFocus}
       options={options}
+      open={true}
```

Comment:
@chrispulsinelli-okta We forgot to remove this 😄

Reply:
You mean people want to close this?!

Fixing it :)